erpnext/selling/doctype/sales_order/sales_order.js

```diff
@@ -251,6 +283,9 @@ erpnext.selling.SalesOrderController = erpnext.selling.SellingController.extend(
 						},
 						primary_action_label: __('Make')
 					});
+					d.$wrapper.find("div[data-fieldname=\"sales_order_item\"]").hide();
```

Comment:
God save us

Reply:
🤢 Yikes... why even include the fields in the dialog definition if they are being hidden? am I missing something?

Reply:
@sushantnadkar says that if you set it as "hidden" the value doesn't show up in the DOM, and he uses that to check which child table items are ticked